qiskit/circuit/classical/types/types.py

```diff
@@ -19,11 +19,7 @@
 
 from __future__ import annotations
 
-__all__ = [
-    "Type",
-    "Bool",
-    "Uint",
-]
+__all__ = ["Type", "Bool", "Float", "Uint"]
```

Comment:
No need to change it back, but I _think_ I deliberately put in the trailing comma because this is a list that's expected to grow.